### PR TITLE
Add max-compact-buffers feature flag.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java [hunk 1]
```diff
@@ -1026,6 +1026,22 @@ public void default_distributor_three_phase_update_config_controlled_by_properti
...
+        ProtonConfig.Builder protonBuilder = new ProtonConfig.Builder();
+        cc.getSearch().getConfig(protonBuilder);
+        ProtonConfig protonConfig = new ProtonConfig(protonBuilder);
+        assertEquals(1, protonConfig.documentdb().size());
+        return protonConfig.documentdb(0).allocation().max_compact_buffers();
+    }
+
+    @Test
+    public void default_max_compact_buffers_config_controlled_by_properties() {
+        assertEquals(2, resolveMaxCompactBuffers(2));
```
Comment (geirst) [inline config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java:1041]: Consider adding a test case for the default value of 1 (where maxCompactBuffers is not specified in TestProperties).

**Review by geirst — APPROVED** [review]
👍

File: config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java (hunk 1 above)
Comment (toregge) [reply config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java]: Fixed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**